app/src/main/java/org/oppia/android/app/drawer/ExitProfileDialogFragment.kt

```diff
@@ -43,6 +53,15 @@ class ExitProfileDialogFragment : DialogFragment() {
       false
     )
 
+    val isAdminSelected = args.getBoolean(
```

Comment:
we can use `isAdministratorControlsSelected` as `isAdminSelected ` feels more like Admin profile is selected or not?

Reply:
Done